sdk/python/kfp/dsl/databricks/_cluster_op.py

```diff
@@ -0,0 +1,221 @@
+from .._resource_op import ResourceOp
+
+class CreateClusterOp(ResourceOp):
+    """Represents an op which will be translated into a Databricks Cluster creation resource
```

Comment:
Should we write `Op` rather than `op` here?

Reply:
If so, you may want to search/replace for them all

Reply:
Done